package.json

```diff
@@ -1,99 +1,100 @@
 {
```

Comment:
why has all the indentation changed in this file...

Reply:
I edited the file to add the `install-functions` script, I'm assuming that prettier converts tabs to spaces for `.json` files